src/api/edge/imagesInfo.js

```diff
@@ -0,0 +1,116 @@
+export const EDGE_API = '/api/edge/v1';
+import { instance } from '@redhat-cloud-services/frontend-components-utilities/interceptors/interceptors';
```

Comment:
This shouldn't be used directly like this. 

The axios instance returned by this will have interceptors attached that will use `chrome` functions via the window object. 

Instead the `useAxiosWithPlatformInterceptors` should be used, which will use `chrome` functions via the `useChrome` hook.